src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2740,6 +2774,138 @@ public void NullExceptionInLabeledStatement()
+                // else {}
+                Diagnostic(ErrorCode.ERR_SyntaxError, "else").WithArguments("(", "else").WithLocation(1, 1),
+                // (1,1): error CS1525: Invalid expression term 'else'
+                // else {}
+                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "else").WithArguments("else").WithLocation(1, 1),
+                // (1,1): error CS1026: ) expected
+                // else {}
+                Diagnostic(ErrorCode.ERR_CloseParenExpected, "else").WithLocation(1, 1),
+                // (1,1): error CS1525: Invalid expression term 'else'
+                // else {}
+                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "else").WithArguments("else").WithLocation(1, 1),
+                // (1,1): error CS1002: ; expected
+                // else {}
+                Diagnostic(ErrorCode.ERR_SemicolonExpected, "else").WithLocation(1, 1)
+                );
+            N(SyntaxKind.IfStatement);
+            {
+                M(SyntaxKind.IfKeyword);
+                M(SyntaxKind.OpenParenToken);
+                M(SyntaxKind.IdentifierName);
+                {
+                    M(SyntaxKind.IdentifierToken);
+                }
+                M(SyntaxKind.CloseParenToken);
+                M(SyntaxKind.ExpressionStatement);
+                {
+                    M(SyntaxKind.IdentifierName);
+                    {
+                        M(SyntaxKind.IdentifierToken);
+                    }
+                    M(SyntaxKind.SemicolonToken);
+                }
+                N(SyntaxKind.ElseClause);
+                {
+                    N(SyntaxKind.ElseKeyword);
+                    N(SyntaxKind.Block);
+                    {
+                        N(SyntaxKind.OpenBraceToken);
+                        N(SyntaxKind.CloseBraceToken);
+                    }
+                }
+            }
+            EOF();
+        }
+
+        [WorkItem(27866, "https://github.com/dotnet/roslyn/issues/27866")]
+        [Fact]
+        public void ParseSubsequentElseWithoutPrecedingIfStatement()
+        {
+            UsingStatement("if (a) { } else { } else { }",
+                // (1,1): error CS1073: Unexpected token 'else'
+                // if (a) { } else {} else {}
+                Diagnostic(ErrorCode.ERR_UnexpectedToken, "if (a) { } else { }").WithArguments("else").WithLocation(1, 1)
+                );
+            N(SyntaxKind.IfStatement);
+            {
+                N(SyntaxKind.IfKeyword);
+                N(SyntaxKind.OpenParenToken);
+                N(SyntaxKind.IdentifierName);
+                {
+                    N(SyntaxKind.IdentifierToken, "a");
+                }
+                N(SyntaxKind.CloseParenToken);
+                N(SyntaxKind.Block);
+                {
+                    N(SyntaxKind.OpenBraceToken);
+                    N(SyntaxKind.CloseBraceToken);
+                }
+                N(SyntaxKind.ElseClause);
```

Comment:
>ElseClause [](start = 29, length = 10)

The second `else { }` isn't captured, since this code parses as two statement. You may need to wrap the whole thing in a block `{ ... }` or use `UsingTree` instead of `UsingStatement`.